editor/store/effects.js

```diff
@@ -118,12 +118,23 @@ export default {
 
 		let request;
 		if ( isAutosave ) {
-			toSend.parent = post.id;
+			const autosaveData = {};
+
+			// Ensure autosaves contain all fields.
+			Object.assign( autosaveData,
+				{
+					title: post.title,
+					content: post.content,
```

Comment:
Unlike `title` and `excerpt`, `toSend` will always have an assigned value for `content`, and thus will always override. This line will never take effect.

Reply:
Also wondering: Since `isEditedPostAutosaveable` uses property values from `state.autosave` to consider a field as having been changed, should we be using those here as well as the original value to fall back to? Maybe with a `getCurrentAutosave` selector?

```js
toSend = {
	...getCurrentAutosave(),
	...toSend,
	parent: post.id
};
```